app/src/main/java/io/homeassistant/companion/android/sensors/TimeZoneManager.kt

```diff
@@ -49,18 +49,17 @@ class TimeZoneManager : SensorManager {
             return
 
         val timeZone: TimeZone = TimeZone.getDefault()
-        val currentZone = timeZone.getDisplayName(Locale.ENGLISH)
-        val date = Date()
+        val currentlyInDst = timeZone.inDaylightTime(Date())
 
         onSensorUpdated(
             context,
             currentTimeZone,
-            currentZone,
+            timeZone.getDisplayName(currentlyInDst, TimeZone.LONG, Locale.getDefault()),

```

Comment:
Will the state still be in english if the user had a non english language selected? It looks like this will return the display name in whatever locale is selected and I believe core expects the state to be in english.

Reply:
No it will change to match the locale, that's kind of the point (and the reason I listed this as a breaking change for non-English users). Or was there a reason for always using English?

Core seems to treat the state like a string so any value is accepted? I've tested this and it seems to work fine, nothing in the logs either.

Reply:
For states core wants them provided in English and then the integration can provide the translations so they appear correct in the frontend. There's an ADR that covers this. Keeping it all in English also provides some consistency too.

https://github.com/home-assistant/architecture/blob/master/adr/0009-translations-2.0.md

Reply:
That's the guideline I was looking for, thanks!